### PR TITLE
Data Privacy: First draft

### PR DESCRIPTION
**Pull request by ewels**
Added a new page called "Data Privacy" which outlines the metadata collected by Tower about runs.

Based on an earlier Google Docs file that @evanfloden sent - this will likely need some updating. We can also think about rephrasing and whether all of this detail is necessary.

Feedback / edits welcome!

### CODE REVIEW

File: docs/data-privacy/overview.md [hunk 1]
```diff
@@ -0,0 +1,445 @@
...
+ }
+```
+
+`HTTP url=https://api.tower.nf/trace/workflow; payload:`
+
+```json
+ {
+     "workflow": {
+         "start": "2020-02-14T14:06:54.028+01:00",
+         "projectDir": "/Users/pditommaso/.nextflow/assets/nextflow-io/hello",
```
Comment (pditommaso) [inline docs/data-privacy/overview.md:164]: Maybe better to anonymise these

**Comment by abhi18av** [comment]
@ewels , I've changed the base to `staging` so that as soon as this is merged a preview would be deployed automatically and then if there's any need we can tweak and publish.

File: docs/data-privacy/overview.md (hunk 1 above)
Comment (ewels) [reply docs/data-privacy/overview.md:164]: Will do!

**Comment by abhi18av** [comment]
No worries @ewels , let me catch up with Graham and Ben once - I see that there has been a lot of activity recently and perhaps we can simplify the instructions for facilitating contributions.

**Comment by gwright99** [comment]
The flow @bentsherman and I have been using is:
1. Create feature branches off of staging.
2. Do the heavy review on the `feature -> staging` assignment of the PR review
3. Squash and merge to `staging` once approved.
4. Rubberstamp the `staging -> master` PR.

Looks like @abhi18av got everything sorted from a base branch perspective.

**Comment by ewels** [comment]
Sounds good! I branched from `staging` so should be fine for this flow 👍🏻

Any thoughts about using a similar  Netlify flow as used for the Seqera website? Then we could have deploy previews on all PRs and probably skip `staging`.

**Comment by gwright99** [comment]
The commit to staging automatically creates a netlify preview.
(See DocsOps page for details and links: https://www.notion.so/seqeralabs/DocOps-8af778eb34cd49d8b34fc39a6803a333).

I've personally found it easier to pull the branch locally and use `mkdocs serve`, but am open to new ways of collaboration if I means we can get more eyes and writers into the documentation creation process.

**Comment by ewels** [comment]
Yeah I tested it locally as well (some hiccup with one of the plugins not working with the free version of mkdocs material, but it was otherwise smooth). If I had checked those docs I would have been ok though 🤦🏻 

I guess my question is why not to use the Netlify app with built-in deployment previews to PRs with the Netlify bot? Instead of having separate staging and production instances. See the [latest Seqera website PR I did](https://github.com/seqeralabs/website-ver2/pull/281#issuecomment-1134706421) for an example.

image

For writing it's not that helpful as you have to wait for new builds, but for reviewing PRs it's nice to have a one-click preview in your browser alongside the PR. And presumably simpler infrastructure if there's just one deployment and one workflow of PR to master -> preview deployment -> merge & automatically deploy to prod. Also means that you can preview as many PRs in parallel as you like, no constraint to having just one at a time on staging.

**Comment by ewels** [comment]
Sorry, I know this is all a bit off topic 😅 It's just kind of fun / interesting to swoop in and try to make a small contribution to every website in the Seqera realm. Each one has it's own process.. 😀

**Comment by gwright99** [comment]
@abhi18av for more background - this was they way it worked when I joined. 

Personally, I find I like having the two stage process as I found that's worked well for coordinating amongst various changes being done by different folks, but I'm all for further optimization (if it works for our needs).

**Comment by ewels** [comment]
Yup, no pressure from me at all (if it ain't broke, don't fix it!). But just so you know what exists elsewhere 🙂 I guess @olehpratsko might be able to help set it up on this repo if it's of interest.

I guess that the two are not mutually exclusive, so could presumably have integrated deployment previews on PRs to stage and then also the staging deployment.

**Comment by abhi18av** [comment]
Actually, the backstory for a double website process

- Initially we didn't have enough Netlify quota for build times, so building only the staging website saved those minutes for other Seqera websites. Maybe we have elevated our plan recently if we're building on a per PR basis 🤔 

- The individual PR builds seemed a little too much for a doc website since we're a polar-bear friendly org 😉  (And the main website if updated quite frequently compared to docs) 

- There's actually a two-stage process for publishing the staging one (1) [Github action](https://github.com/seqeralabs/nf-tower-docs/blob/master/.github/workflows/staging-html.yml), which deals with `mike` based Markdown to HTML conversion (2) Netlify publishes the website to a static URL for a sanity check.

- We needed a static URL to be able to test the nuances of `mkdocs` + `mike` for testing the internal links 

That being said, I'd like to mention that I do like the idea of simplifying the entire process and only requiring `mkdocs insiders` locally in case someone would like to test and contribute changes locally, `mike` makes things complicated locally.

Plus, its better to have a consistent process on an org-wide basis.

**Comment by abhi18av** [comment]
I'm moving ahead with the merge for this PR as it's quite relevant for the questions we often receive from clients.

For the simplification of the process, I've created a tracking issue here https://github.com/seqeralabs/nf-tower-docs/issues/211 

Thanks for sharing your thoughts everyone!

**Pull request merged**